### PR TITLE
Docs: update UG

### PR DESCRIPTION
**Pull request by muhd97**

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -78,7 +78,7 @@ Format: `delete <index>`
 === Modify a Recipe — `modify`
 Modifies attributes of a recipe.
 
-Format: `modify <index> [ing/<idx. ingredient, quantity> [ins/instruction; ...] [tags/tag, ...]`
+Format: `modify <index> [ing/<idx. ingredient, quantity> [ins/instruction; ...] [cal/calories] [tags/tag, ...]`
```
Comment (sharadhr) [inline docs/UserGuide.adoc]: remove square brackets around ingredient and instructions; these refer to *optional* arguments.

**Review by sharadhr — DISMISSED** [review]

File: docs/UserGuide.adoc (hunk 1 above)
Comment (muhd97) [reply docs/UserGuide.adoc]: isn't modifying it optional ?

Comment (kevinswk94) [reply docs/UserGuide.adoc]: they are still optional since we may not be modifying the ingredients or instructions.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -78,7 +78,7 @@ Format: `delete <index>`
 === Modify a Recipe — `modify`
 Modifies attributes of a recipe.
 
-Format: `modify <index> [ing/<idx. ingredient, quantity> [ins/instruction; ...] [tags/tag, ...]`
+Format: `modify <index> [ing/<idx. ingredient, quantity> [ins/instruction; ...] [cal/calories] [s/serving size] [r/rating] [tags/tag, ...]`
```
Comment (kevinswk94) [inline docs/UserGuide.adoc]:
```suggestion
Format: `modify <index> [ing/ingredient, quantity; ...] [ins/instruction; ...] [cal/calories] [s/serving size] [r/rating] [tags/tag, ...]`
```

**Review by kevinswk94 — APPROVED** [review]
looks good! accept my suggestion if you think its suitable

**Pull request merged**